homeworks/week03/README.md

```diff
@@ -0,0 +1,266 @@
+# Week 3, Homework
+
+For this homework you will add interactive functionality to a webpage
+that displays a static list of movies.
+
+You will be writing JavaScript code using the jQuery library, and will
+also add some CSS to style the page.
+
+In a way you will be building on the homework assignment from Week 2,
+but to simplify things, we will no longer be working with the
+JavaScript `render()` functions that you wrote last week, and instead
+will start with a file of static HTML. The HTML markup in the file is
+very similar to what you generated last week, but there are some
+differences, and working with a static HTML file that you can examine
+in your text editor will hopefully make things easier than using the
+JavaScript `render()` functions.
+
+The goal will be to handle various events, so that what is displayed
+is dynamically changed by user actions.
+
+As with the previous homework, to begin, download the zip file for
```

Comment:
Do we really need a zip file? There are just a few files, which will be in the github repo, right?